aruco-test/README.md

```
@@ -0,0 +1,17 @@
+git clone https://github.com/luxonis/depthai-core.git && cd depthai-core
+python3 -m venv venv
+source venv/bin/activate
+# Installs library and requirements
+python3 examples/python/install_requirements.py
+
+
+cd examples/python
+# Run YoloV6 detection example
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)